File: sql-data-partitioning/src/main/java/com/mageddo/sqldatapartitioning/dao/SkinPriceNoPtDAO.java
package com.mageddo.sqldatapartitioning.dao;

import com.mageddo.sqldatapartitioning.entity.SkinPriceEntity;

import java.time.LocalDate;

public interface SkinPriceNoPtDAO {
	void create(SkinPriceEntity skinPriceEntity);

	SkinPriceEntity find(LocalDate date, Long id);

}

File: sql-data-partitioning/src/main/java/com/mageddo/sqldatapartitioning/dao/SkinPriceNoPtDAO.kt
package com.mageddo.sqldatapartitioning.dao

import com.mageddo.sqldatapartitioning.entity.SkinPriceEntity

import java.time.LocalDate

interface SkinPriceNoPtDAO {

	fun create(skinPriceEntity: SkinPriceEntity)

	fun find(date: LocalDate, id: Long?): SkinPriceEntity?

}
